### PR TITLE
Add info to PowerToys in mac to windows guide

### PR DESCRIPTION
**Pull request by noraa-junker**
Related issue: #3637

cc: @mattwojo & @crutkas

### CODE REVIEW

**Comment by PRMerger19** [comment]
@Aaron-Junker : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

**Comment by crutkas** [comment]
i'm good if @mattwojo is

**Comment by PRMerger3** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
I added an edit, hopefully for clarity. Let me know if I missed the mark, and I'll revise. Also, I shortened the docs.microsoft.com link to a relative link. Thanks.

File: hub/dev-environment/mac-to-windows.md [hunk 1]
```diff
@@ -19,6 +19,9 @@ For app development, the nearest equivalent to Xcode would be [Visual Studio](ht
 
 ## Keyboard shortcuts
 
+> [!TIP]
+> You can use [PowerToys Keyboard Manager](/windows/powertoys/keyboard-manager) to map Windows shortcuts to the shortcuts you're accustomed to using on a Mac.
```
Comment (noraa-junker) [inline hub/dev-environment/mac-to-windows.md]:
```suggestion
> You can use [PowerToys Keyboard Manager](/windows/powertoys/keyboard-manager) to map Windows shortcuts to the shortcuts you're accustomed to use on a Mac.
```

I'm not a native english speaker, but this seems to be better for me @ktoliver

**Comment by PRMerger3** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
@Aaron-Junker Thanks for your reply. I revised to make the sentence more direct.

From the comments, it looks like we need @mattwojo or @crutkas to add `#sign-off`. Thanks.

File: hub/dev-environment/mac-to-windows.md (hunk 1 above)
Comment (crutkas) [reply hub/dev-environment/mac-to-windows.md]: agreed Aaron here

**Review by crutkas — APPROVED** [review]

**Comment by mattwojo** [comment]
Thanks @Aaron-Junker! Appreciate all the work you've been doing on these docs. Great updates. You rock!

**Comment by noraa-junker** [comment]
> Thanks @Aaron-Junker! Appreciate all the work you've been doing on these docs. Great updates. You rock!

Thank you! 😊

**Pull request merged**